deploy/blobfuse-proxy/README.md

```diff
@@ -1,6 +1,5 @@
 # Blobfuse Proxy
  - supported CSI driver version: v1.6.0+
- - only available on **debian** OS based agent node (not available on OpenShift)
```

Comment:
why it's removed?

Reply:
https://github.com/kubernetes-sigs/blob-csi-driver/issues/891#issuecomment-1522297224

Users may choose not to enable it on mariner while actually it can be used on mariner or other system as well.